grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilderTest.java

```diff
@@ -260,6 +277,113 @@ void enableHttpJsonTranscodingWithoutJsonSupport() {
+        final AtomicInteger responseStreamCallCnt = new AtomicInteger();
+
+        final AtomicInteger requestParseCallCnt = new AtomicInteger();
+        final AtomicInteger responseParseCallCnt = new AtomicInteger();
+
+        final Marshaller<SimpleRequest> customRequestMarshaller = new PrototypeMarshaller<SimpleRequest>() {
+            @Override
+            public Class<SimpleRequest> getMessageClass() {
+                return null;
+            }
+
+            @Nullable
+            @Override
+            public SimpleRequest getMessagePrototype() {
+                return SimpleRequest.getDefaultInstance();
+            }
+
+            @Override
+            public InputStream stream(SimpleRequest value) {
+                requestStreamCallCnt.incrementAndGet();
+                return TestServiceGrpc.getUnaryCallMethod().getRequestMarshaller().stream(value);
+            }
+
+            @Override
+            public SimpleRequest parse(InputStream stream) {
+                requestParseCallCnt.incrementAndGet();
+                return TestServiceGrpc.getUnaryCallMethod().getRequestMarshaller().parse(stream);
+            }
+        };
+
+        final Marshaller<SimpleResponse> customResponseMarshaller = new PrototypeMarshaller<SimpleResponse>() {
+            @Override
+            public Class<SimpleResponse> getMessageClass() {
+                return null;
+            }
+
+            @Nullable
+            @Override
+            public SimpleResponse getMessagePrototype() {
+                return SimpleResponse.getDefaultInstance();
+            }
+
+            @Override
+            public InputStream stream(SimpleResponse value) {
+                responseStreamCallCnt.incrementAndGet();
+                return TestServiceGrpc.getUnaryCallMethod().getResponseMarshaller().stream(value);
+            }
+
+            @Override
+            public SimpleResponse parse(InputStream stream) {
+                responseParseCallCnt.incrementAndGet();
+                return TestServiceGrpc.getUnaryCallMethod().getResponseMarshaller().parse(stream);
+            }
+        };
+
+        final TestServiceImplBase testService = new TestServiceImpl(
+                Executors.newSingleThreadScheduledExecutor());
+        final ServerServiceDefinition serviceDefinition = ServerInterceptors.useMarshalledMessages(
+                testService.bindService(), customRequestMarshaller, customResponseMarshaller);
+
+        final GrpcService service = GrpcService.builder()
+                                               .addService(serviceDefinition)
+                                               .useMethodMarshaller(useMethodMarshaller)
+                                               .build();
+
+        try (Server server = Server.builder()
+                                   .service(service)
+                                   .build()) {
+            server.start().join();
```

Comment:
Question) Is it possible to use a single `Server` via `RegisterExtension` so that we don't have to unnecessarily start/stop the server on each run?

e.g.
```java

static val customRequestMarshaller = ...
static val customResponseMarshaller = ...

@RegisterExtension
static ServerExtension server = ... {
  static val serviceDefinition = ...;
  GrpcService.builder()
    .addService(serviceDefinition)
    ...
}

@BeforeEach
void beforeEach() {
  customRequestMarshaller.reset();
  customResponseMarshaller.reset();
}
```

Reply:
I think, it's possible. I'll try.

Reply:
Applied feedbacks, Thanks.